### ISSUE TITLE
Weight TMDB ratings by number of reviews

### ISSUE THREAD

**Issue by ryanwkirby** [sct/overseerr#3868]
### Description

I'm always frustrated when trying to sort movies by average TMDB score, because all the small movies with 1 rating of "100%" get mixed up at the top. 

### Desired Behavior

Use both data points (rating & review count) to determine TMDB score. 

Explanation: 
https://www.evanmiller.org/how-not-to-sort-by-average-rating.html

Basically just use the lower end of a confidence interval.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by rjdoug** [comment]
Can confirm this does my head in. Do I need to wait until out of triage before working on this?

**Comment by marchershey** [comment]
> Can confirm this does my head in. Do I need to wait until triaged before working on this?

No. Github Issues are basically ignored in this repo. I recommend you to work on it, and when finished, create a pull request. You'll get the attention then.

**Comment by HWiese1980** [comment]
This has to be done. The current sorting by TMDB rating is completely useless because it puts media that 1/1 user finds great before media that 99/100 users find great.

**Comment by ryanwkirby** [comment]
bump
